Cargo.toml

```diff
@@ -4,13 +4,19 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+aws-config = "0.55"
```

Comment:
Had dependency conflicts so managing these manually is required before an aws 1.0 sdk ships.